--- Shdr.java	2020-09-16 14:19:38.000000000 +0200
+++ new/Shdr.java	2020-09-16 14:33:32.622235003 +0200
@@ -3,12 +3,9 @@
 
 package jogamp.common.os.elf;
 
-import java.nio.*;
-
-import com.jogamp.gluegen.runtime.*;
-import com.jogamp.common.os.*;
-import com.jogamp.common.nio.*;
-import jogamp.common.os.MachineDataInfoRuntime;
+import com.jogamp.common.nio.Buffers;
+import com.jogamp.common.nio.StructAccessor;
+import com.jogamp.common.os.MachineDataInfo;
 
 
 public class Shdr {
